File: src/main/java/com/example/calendarexperiments/jackson/BitSetModule.java
package com.example.calendarexperiments.jackson;

import com.fasterxml.jackson.databind.module.SimpleModule;

import java.util.BitSet;

public class BitSetModule extends SimpleModule {
	public BitSetModule() {
		super(BitSetModule.class.getSimpleName());
		addSerializer(new BitSetSerializer());
		addDeserializer(BitSet.class, new BitSetDeserializer());
	}
}

File: src/main/kotlin/com/example/calendarexperiments/bitset/jackson/BitSetModule.kt
package com.example.calendarexperiments.bitset.jackson

import com.fasterxml.jackson.databind.module.SimpleModule
import java.util.*

class BitSetModule : SimpleModule(BitSetModule::class.java.simpleName) {
    init {
        addSerializer(BitSetSerializer())
        addDeserializer(BitSet::class.java, BitSetDeserializer())
    }
}